pytket/phir/phirgen_parallel.py

```diff
@@ -280,6 +280,31 @@ def format_and_add_primary_commands(
             groups2qops(fmt_g2q, ops)
 
 
+def adjust_phir_transport_time(ops: list["JsonDict"], machine: "Machine") -> None:
+    """Analyze the generated phir and adjust the transport time."""
+    adjustment = 0.0
+    for op in ops:
+        if "qop" in op:
+            match op["qop"]:
+                case "RZ" | "R1XY":
+                    adjustment += machine.sq_time
+                case "RZZ":
+                    adjustment += machine.tq_time
+                case "Measure":
+                    adjustment += machine.meas_prep_time
+        if "block" in op and op["block"] == "qparallel":
+            first_op = op["ops"][0]["qop"]
+            match first_op:
+                case "RZ" | "R1XY":
+                    adjustment += machine.sq_time
+                case "RZZ":
+                    adjustment += machine.tq_time
```

Comment:
Can there be any other case here? If so, we should handle that/include a warning.